build/_k-standard-goals.shade

```diff
@@ -95,10 +95,16 @@ default Configuration='${E("Configuration")}'
         var msbuildPath =  Path.Combine(programFilesX86, "MSBuild", msbuildVersions[i], "Bin", "MSBuild.exe");
         if (File.Exists(msbuildPath))
         {
+          var commonParameters =
+            " /p:Configuration=" + Configuration +
+            " /p:ProductVersion=1.0.0" +
```

Comment:
Hmm I thought in general the project.json files had the version in it, though they have `x.0.0-*` and then the `*` comes from the build system. Local builds have `*` == `t<timestamp>`, whereas the CI has `*` == `<release quality>` e.g. beta4, rc1, etc. As such, I don't think there's a global way of doing this.

Now, it could very well be that it so happens that all our native projects are all 1.0.0 right now so it's technically correct.